test/testcases/test_data_onboarding.py

```diff
@@ -44,3 +44,38 @@ def test_proto_data_onboarding(self, setup, test_scenario, test_input, expected)
                                           password=setup["splunk_password"])
         logger.info("Splunk received %s events in the last hour", len(events))
         assert len(events) == expected
+
+    @pytest.mark.parametrize("test_scenario, test_input, expected", [
```

Comment:
What do you think about not using `pytest.mark.parametrize` if you have just 1 thing to parametrize? Here and below.